doc/classes/Environment.xml

```diff
@@ -9,7 +9,11 @@
 		- Glow
 		- Tonemap (Auto Exposure)
 		- Adjustments
-		These effects will only apply when the [Viewport]'s intended usage is "3D" or "3D Without Effects". This can be configured for the root Viewport with [member ProjectSettings.rendering/quality/intended_usage/framebuffer_allocation], or for specific Viewports via the [member Viewport.usage] property.
+		In "2D Without Sampling", all post-processing effects will be turned off. In "3D Without Effects", these options will be invalid:
+		- Ssao
+		- Ss reflections
+		This can be done using [member ProjectSettings.rendering/quality/intended_usage/framebuffer_allocation] or via the [member Viewport.usage] property to configure the root viewport for a specific viewport.
+		Note that [member ProjectSettings.rendering/quality/intended_usage/framebuffer_allocation.mobile] uses "3D Without Effects" by default, which will improve the performance on mobile devices, but at the same time affect the screen display on mobile devices.
```

Comment:
This reference seems not to be supported by `make_rst.py`:
```
ERROR: Bad reference: 'ProjectSettings.rendering/quality/intended_usage/framebuffer_allocation.mobile', file: Environment
```
This could maybe be fixed in `make_rst.py` after making sure that it's also supported by the engine EditorHelp itself (CC @pycbouh), but in the meantime the simplest might be to rephrase:
```suggestion
		Note that [member ProjectSettings.rendering/quality/intended_usage/framebuffer_allocation] has a mobile platform override to use "3D Without Effects" by default. It improves the performance on mobile devices, but at the same time affects the screen display on mobile devices.
```

Reply:
> This reference seems not to be supported by `make_rst.py`:
> 
> ```
> ERROR: Bad reference: 'ProjectSettings.rendering/quality/intended_usage/framebuffer_allocation.mobile', file: Environment
> ```
> 
> This could maybe be fixed in `make_rst.py` after making sure that it's also supported by the engine EditorHelp itself (CC @pycbouh), but in the meantime the simplest might be to rephrase:

My git usage really makes me feel like a git
Is now okay?

Reply:
The changes are good, now you need to rebase the branch to squash those two commits into one.
In the merged Git history, we don't want commits like "Apply suggestions from code review", it's better if those changes are retrofitted into the original commit which explains what it does. Here's how you can rebase the PR: https://docs.godotengine.org/en/latest/community/contributing/pr_workflow.html#the-interactive-rebase